frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx

```diff
@@ -117,6 +117,17 @@ export const TableNode: FC<Props> = ({
                           styles.handleCardinalityHighlighted,
                       ])}
                     />
+                    <span
+                      className={clsx([
+                        styles.handleCardinalityComment,
+                        styles.handleCardinalityCommentRight,
+                        (isHighlighted ||
+                          highlightedHandles?.includes(handleId)) &&
+                          styles.handleCardinalityCommentHighlighted,
+                      ])}
+                    >
+                      1
```

Comment:
I believe this "1" should actually be "0 or 1". However, I've omitted the "0" because displaying it would make the information too verbose.
![ss 2487](https://github.com/user-attachments/assets/d8457826-5d22-47b6-8d55-507368ec5085)

Reply:
Agreed, it was good to focus on simplicity rather than strictness.

---

drawdb is also in `1` notation:

<img width="793" alt="スクリーンショット 2024-12-13 16 13 35" src="https://github.com/user-attachments/assets/99f7d9ad-63dc-4d67-9f37-853394f90c17" />